configs/components/hiera.rb

```diff
@@ -9,23 +9,29 @@
 
   pkg.replaces 'pe-hiera'
 
+  flags = "--bindir=#{settings[:bindir]} \
+           --sitelibdir=#{settings[:ruby_vendordir]} \
+           --ruby=#{File.join(settings[:bindir], 'ruby')}"
+
+  if platform.is_windows?
+    pkg.add_source("file://resources/files/windows/hiera.bat", sum: "bbe0a513808af61ed9f4b57463851326")
+    pkg.install_file "../hiera.bat", "#{settings[:link_bindir]}/hiera.bat"
+    flags = "--bindir=#{settings[:hiera_bindir]} \
+             --sitelibdir=#{settings[:hiera_libdir]} \
+             --ruby=#{File.join(settings[:ruby_dir], 'bin/ruby')}"
+  end
+
   pkg.install do
     ["#{settings[:host_ruby]} install.rb \
-    --ruby=#{File.join(settings[:bindir], 'ruby')} \
-    --bindir=#{settings[:bindir]} \
-    --configdir=#{settings[:puppet_codedir]} \
-    --sitelibdir=#{settings[:ruby_vendordir]} \
     --configs \
-    --quick \
```

Comment:
yep, it's not even an option in this install script, I'm not sure how this was working before but theres no quick option under https://github.com/puppetlabs/hiera/blob/master/install.rb#L99L122